medusa/metadata/wdtv.py

```diff
@@ -162,12 +150,11 @@ def get_season_poster_path(show_obj, season):
                 break
 
         if not season_dir:
-            logger.log(u'Unable to find a season directory for season {season_num}'.format
-                       (season_num=season), logger.DEBUG)
+            log.debug(u'Unable to find a season directory for season {0}', season)
             return None
 
-        logger.log(u'Using {path}/folder.jpg as season dir for season {season_num}'.format
-                   (path=season_dir, season_num=season), logger.DEBUG)
+        log.debug(u'Using {path}/folder.jpg as season dir for season {x}',
+                  {'path': season_dir, 'x': season})
```

Comment:
I would agree with you if it was being used in a context where we want to represent a mathematical variable. In that case, ```x``` would fit perfectly. In this context, I don't think it's the best fit as we are just representing a number. I also think it fits better to the example you made. Calling it ```number``` makes most sense to me and imo it's also clearer:
```
for season {number}
```
```
log.debug(u'Using {path}/folder.jpg as season dir for season {number}'
```
In the end it's mostly a metter of personal preference in this case, which is why I didn't request changes.